components/src/components/organisms/Dialog/Dialog.tsx

```diff
@@ -36,19 +43,21 @@ const StyledCard = styled.div(
   ({ theme }) => css`
     display: flex;
     flex-direction: column;
-    align-items: stretch;
-    gap: ${theme.space['6']};
-
-    padding: ${theme.space['6']};
+    align-items: center;
+    gap: ${theme.space['4']};
+    padding: ${theme.space['4']};
```

Comment:
dont we still use 24px padding on mobile dialogs?

Reply:
https://www.figma.com/file/iP5p7Vd2C1VhRjTXizfERU/Thorin?node-id=690%3A9627&t=AEB5HNkT7r4o8IIq-0
16px on mobile
24px on desktop

Reply:
i guess the transaction modal has different padding then... weird

Reply:
I think it's been inconsistently applied. I asked for standardized format yesterday at the meeting.